src/cms.js

```diff
@@ -5,12 +5,12 @@ const forge = require('node-forge')
 const util = require('./util')
 
 class CMS {
```

Comment:
What does CMS stand for?

Reply:
PKCS #7: Cryptographic Message Syntax; [rfc 2315](https://tools.ietf.org/html/rfc2315]